### PR TITLE
Update README with Flathub info

### PR DESCRIPTION
**Pull request by mbugni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,10 +108,11 @@ The executable was built in Kubuntu 22.04 LTS so there is a chance that it will
 
 #### Install via Flatpak
 
-Thanks to [Massi aka Ergosum](https://github.com/mbugni), Doom Runner can now be installed via [Flatpak](https://flatpak.org/) from this package TODO: link, using the following commands:
```
Comment (Youda008) [inline README.md:111]: You don't wish to be listed as author? Ok.

File: README.md [hunk 2]
```diff
@@ -108,10 +108,11 @@ The executable was built in Kubuntu 22.04 LTS so there is a chance that it will
...
-```
-TODO
-```
+Doom Runner can be installed via [Flatpak](https://flatpak.org/) from this link:
+
+[![Download Doom Runner](https://dl.flathub.org/assets/badges/flathub-badge-en.png 'Download Doom Runner')](https://flathub.org/apps/io.github.Youda008.DoomRunner)
```
Comment (Youda008) [inline README.md:113]: Was it intended to display this big button or is it accidental?
obrazek

File: README.md (hunk 1 above)
Comment (mbugni) [reply README.md:111]: It's not a problem, the important is that now it's available to download.

File: README.md (hunk 2 above)
Comment (mbugni) [reply README.md:113]: It's not accidental, is an [official badge](https://flathub.org/badges). A white version is available too.

Comment (Youda008) [reply README.md:113]: ok, good

**Comment by Youda008** [comment]
Thank you!

**Pull request merged**